config.py

```diff
@@ -0,0 +1,10 @@
+from os import environ
+
+
+class Config:
+    # TESTING = environ.get('TESTING')
```

Comment:
What's the situation with these commented-out bits? Consider adding a comment capturing what they would do if uncommented and when/whether you think that will need to happen.

Reply:
We're gonna fix it later. I should add a comment, but I'm lazy.

Reply:
#32